test/models/User.spec.js

```diff
@@ -0,0 +1,94 @@
+import assert from 'power-assert';
+import mockgoose from 'mockgoose';
+import Promise from 'bluebird';
+import UserModel from 'models/User';
+
+describe('/models/User', ()=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).